### DISCUSSION TITLE
M2O behaviour in app

### DISCUSSION THREAD

**Discussion by mmmaxxxe** [directus/directus#13085] [Archive]
### Describe the Bug

There is a different behaviour in M2o from view to edit mode after updating from 9.8 to 9.9
Tried to update to 9.9.1 without luck

VIEW (correct):

![image](https://user-images.githubusercontent.com/102175921/166242626-0d92980a-8a12-4805-8639-7147a63a8cf3.png)

EDIT (incorrect)

![image](https://user-images.githubusercontent.com/102175921/166242965-8f436018-868d-4d01-bec8-6782ac60a055.png)

### To Reproduce

Upgrade an M2O from 9.8 to 9.9

### Errors Shown

None

### What version of Directus are you using?

9.9.0

### What version of Node.js are you using?

v14.15.4

### What database are you using?

Postgres 12.9 (Ubuntu 12.9-0ubuntu0.20.04.1)

### What browser are you using?

chrome

### How are you deploying Directus?

Digitalocean one click app

**Comment by rijkvanzanten** [comment]
Could you share the configuration of these relationships?

**Reply by mmmaxxxe** [reply]
![image](https://user-images.githubusercontent.com/102175921/166247947-1aef7bfe-797d-40e7-be3e-6f48573bc089.png)

![image](https://user-images.githubusercontent.com/102175921/166248013-c2d9f7af-25a2-474e-b466-a19af21bad50.png)

![image](https://user-images.githubusercontent.com/102175921/166248093-2ca029a3-a436-449c-980e-ad40400c1c9e.png)

![image](https://user-images.githubusercontent.com/102175921/166248157-54f421ee-7c71-43d3-82ab-3817b63995e6.png)

![image](https://user-images.githubusercontent.com/102175921/166248242-58a22e0d-ed13-4d06-b17d-3f4e6fd781b6.png)

![image](https://user-images.githubusercontent.com/102175921/166248333-c915bdb7-c182-452b-a29a-bfe8f982a17c.png)

![image](https://user-images.githubusercontent.com/102175921/166248396-87a70397-eb1e-44c8-bfd8-b2cca4bc0a1a.png)

**Reply by rijkvanzanten** [reply]
Would you be able to share a database dump / schema snapshot with a minimal reproduction?
